devicetypes/smartthings/testing/simulated-window-shade.src/simulated-window-shade.groovy

```diff
@@ -0,0 +1,183 @@
+/**
+ *  Copyright 2018 SmartThings
```

Comment:
2019

Reply:
I added in 2019; the DTH has been around, at least in my accounts, since about 2018. :)